rdrf/report/schema.py

```diff
@@ -104,6 +104,12 @@ def cde_resolver(cdes, _info, cde_model):
                     value = cde_value["value"]
                     if datatype == 'lookup':
                         return get_widget_class(cde_model.widget_name).denormalized_value(value)
+                    elif cde_model.pv_group:
```

Comment:
So nobody would find using the codes valuable in some circumstances?

Reply:
Potentially. I could add a field to the report designer to capture which one to use. But should I check first with Yvette & Megan if there is a use case for it atm?

Reply:
Yes please, just double check. From discussion today, it seemed like neither did.

Reply:
@ScriptSmith , I have checked with Yvette and Megan and neither could think of a valid use case for requiring the PVG codes.